sandbox/extract-dataset-metadata_emp.R

```diff
@@ -88,4 +89,324 @@ image_count = extract_image_count(exif)
 
 image_count
 
+#### file_size ####
+
+# Total file size of all images in set. Units: GB.
+
+# BEGIN FUNCTION CODE
+
+file_size = sum(exif$FileSize)
+
+# END FUNCTION CODE
+
+# turn code into a function
+
+extract_file_size = function (exif) {
+
+  file_size = sum(exif$FileSize) / 1000000000
+
+  return(file_size)
+
+}
+
+# test function on exif data
+
+file_size = extract_file_size(exif)
+
+file_size
+
+#### percent_images_rtk ####
+
+# Percent of images with an RTK fix. Use the EXIF RTKFlag. Will need to look up what the different codes mean.
+
+# BEGIN FUNCTION CODE
+
+exif$rtk_fix = extract_rtk_fix(exif)
+
+percent_images_rtk = (sum(exif$rtk_fix == TRUE) / nrow(exif)) * 100
+
+# END FUNCTION CODE
+
+# turn code into a function
+
+extract_percent_images_rtk = function (exif) {
+
+  exif$rtk_fix = extract_rtk_fix(exif)
```

Comment:
You don't have to save this back to the `exif` data frame as a new column, you can just create a new variable `rtk_fix`, since this function will not actually affect the data frame outside the function.